web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -99,19 +98,34 @@ const MainLayout = ({ children }) => {
                 <Svg name="plus" className="text-gray-700 w-full h-full" />
               )}
             </li>
-            <li className="h-10 w-10 border-1 rounded-full border-indigo-300 text-indigo-200">
```

Comment:
border got changed from border-2 to border-1, which isn't actually a tailwind class.